packages/connection/legacy/class.jetpack-xmlrpc-server.php

```diff
@@ -577,7 +648,7 @@ function verify_action( $params ) {
 	 *
 	 * @return WP_User|bool
 	 */
-	function login() {
+	public function login() {
 		Jetpack::init()->require_jetpack_authentication();
```

Comment:
This got refactored.

```suggestion
		Jetpack::connection()->require_jetpack_authentication();
```